modules/configuration/src/main/java/org/apache/ignite/internal/configuration/util/ConfigurationUtil.java

```diff
@@ -37,6 +36,9 @@
 
 /** */
 public class ConfigurationUtil {
+    /** Configuration that does nothing but copy of the configuration value. */
```

Comment:
nothing but + noun means "a lot of", which is not applicable here. Maybe something simple, like `Configuration source that copies values`, is better.